src/codec/binary.rs

```diff
@@ -12,6 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! The `binary` codec marshalls raw binary data data as a tremor bytes literal value.
+//!
+//! ## Considerations
+//!
+//! Binary values are always network ( big ) endian
+
```

Comment:
Maybe we should add a note about what binary format the binary codec makes out of e.g. a json record?

Reply:
what happens when you send an event like the following to a connector configured with this codec is what i am hinting at. This is totally unclear from those docs.

```js
{
  "snot": "badger"
}
```

Reply:
I'm not sure how to express it more clearly then `raw binary data` do you have a suggestion?

Reply:
How does the `binary` codec turn an event Value like `{"snot": "badger"}` into raw binary data?
I know that it turns anything coming in into raw binary data. But there might be the case where you have some record from another connector or from wherever. So the case where the `binary` codec receives an event payload with a payload that is not binary. What happens in this case?

Reply:
Ohhhh now I understand the question :D, I thought the other way around. Fixed it 👍 added a note:

```rust
//! When data isn't already represented as `bytes` it will be encoded as JSON document.
```